charts/tidb-cluster/values.yaml

```diff
@@ -175,6 +175,16 @@ tidb:
     exposeStatus: true
     # annotations:
       # cloud.google.com/load-balancer-type: Internal
+  # separateSlowLog: true
+  slowLogTailer:
+    image: busybox:1.26.2
+    resources:
+      limits: {}
+      #   cpu: 50m
+      #   memory: 10Mi
+      requests: {}
+      #   cpu: 50m
```

Comment:
We should specify a request. Very few resources should be needed for this.

Reply:
Is cpu 20m and memory 5MB a good default for both limit and request? (I've tested locally and seems that 10m cpu, 1MB memory is fairly enough)

Reply:
OK, LGTM.

Reply:
I would not use a limit or make the limit a bit higher at least until we have observed it in production..

Reply:
Agreed, maybe 50Mi memory limit is enough. This is pretty small comparing to TiDB itself.